types/stylelint/index.d.ts

```diff
@@ -858,11 +858,24 @@ declare namespace stylelint {
 		quiet?: boolean;
 		quietDeprecationWarnings?: boolean;
 		validate?: boolean;
+		suppressAll?: boolean;
+		suppressRule?: string[];
+		suppressionsLocation?: string;
+		pruneSuppressions?: boolean;
 	};
 
 	/** @internal */
```

Comment:
@ybiquitous

Should we instead use `@experimental` for the ones exported in `index.d.mts`?
It seems contradictory to me.